lib/hammer_cli_foreman/user.rb

```diff
@@ -57,6 +58,8 @@ def self.included(base)
         base.option '--default-location', 'DEFAULT_LOCATION_NAME', _("Default location name")
         base.option "--ask-password", "ASK_PW", " ",
                     :format => HammerCLI::Options::Normalizers::Bool.new
+        base.option '--disabled', 'USER_DISABLED', ' ',
+                    :format => HammerCLI::Options::Normalizers::Bool.new
```

Comment:
Actually this is not needed unless there is no such param in API doc or you want to apply custom changes before sending the request.

The `--disabled` option will be created automatically after `hammer -r` if apidoc cache was updated.